daphne_service_utils/dapf/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+use std::{io::Cursor, path::Path};
+
+use anyhow::{anyhow, Context};
+use daphne::{
+    hpke::HpkeConfig,
+    messages::{decode_base64url_vec, HpkeConfigList},
+    DapVersion,
+};
+use daphne_service_utils::http_headers;
+use prio::codec::Decode;
+use reqwest::Client;
+use url::Url;
+use webpki::{EndEntityCert, ECDSA_P256_SHA256};
+use x509_parser::pem::Pem;
+
+pub trait HttpClientExt {
```

Comment:
Ext is the idiomatic way to define an extension trait.

This trait is only used to add new methods to an external type (`reqwest::Client`). It's not supposed to be implemented for more types (hence the `Ext` suffix)

tbh I want to be a little bit fancy and add methods to HttpClient so that I can do stuff like [this](https://github.com/cloudflare/daphne/blob/12daef8e518a812f288552c968957ccf4b607236/daphne_service_utils/dapf/src/main.rs#L143-L144)